include/TrajectoryCalculator.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include <vector>
+#include <functional>
+
+using namespace std;
+
+class TrajectoryCalculator {
+	std::vector<double> xAxisCoordinates;
+	std::vector<double> yAxisCoordinates;
+
+public:
+	//Getters
+	std::vector<double> getXAxisCoordinates() const;
+	std::vector<double> getYAxisCoordinates() const;
+	
+
+	void CalculateData(
+		double ballVelocity,
+		double firingAngle,
+		double ballRadius,
+		double ballMass,
+		double gravitationalAcceleration,
+		double windVelocity,
+		double windAngle,
+		double atmosphericDensity
+	);
```

Comment:
do tej metody i w sumie innych też, przydało by się je opisać co robią oraz dodać w opisie jednostki do parametrów
jakiś mniej-więcej taki opis zrobić
```suggestion
    /**
     * @brief  Brief description of this function
     * 
     * Long description
     * 
     * @param  value1:  description
     * @param  value2:  description
     * 
     * @return description (tu akurat ta metoda niczego nie zwraca, ale było w przykładzie internetowym)
     */
	void CalculateData(
		double ballVelocity,
		double firingAngle,
		double ballRadius,
		double ballMass,
		double gravitationalAcceleration,
		double windVelocity,
		double windAngle,
		double atmosphericDensity
	);
```

Reply:
trzeba by jeszcze przedyskutować, czy lepiej w plikach .h czy w .cpp te docstringi robić